.github/workflows/pr-cleanup.yaml

```
@@ -0,0 +1,29 @@
+name: Clean up npm pr tags on release
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  release-pr:
+    if: github.event.pull_request.merged == true
+    name: "Clean up npm pr tags on release"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Ensure NPM access
+        run: npm whoami
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Unpublish the package with the tag @pr-${{ github.event.pull_request.number }} from npm
+        id: unpublish
+        run: |
+          echo ::set-output name=version::$(npm --force unpublish pr-${{ github.event.pull_request.number }})
```

I guess this could be trimmed to:

```suggestion
        run: npm --force unpublish pr-${{ github.event.pull_request.number }}
```